fix build failure on arm, this is included in upstream already
patch comes from the maintainer of lcoco Mike Pall <mikelu-0712@mike.de>
---
 src/libs/lua51/lcoco.c |    3 +++
 1 file changed, 3 insertions(+)

Index: luatex-0.20.2/src/libs/lua51/lcoco.c
===================================================================
--- luatex-0.20.2.orig/src/libs/lua51/lcoco.c	2007-12-20 17:46:15.000000000 +0100
+++ luatex-0.20.2/src/libs/lua51/lcoco.c	2007-12-20 17:46:24.000000000 +0100
@@ -286,6 +286,9 @@
 #elif defined(__arm__) || defined(__ARM__)
 
 #if __GLIBC__ == 2 || defined(__UCLIBC__)	/* arm-linux-glibc2 */
+#ifndef __JMP_BUF_SP
+#define __JMP_BUF_SP   ((sizeof(__jmp_buf)/sizeof(int))-2)
+#endif
 #define COCO_PATCHCTX(coco, buf, func, stack, a0) \
   buf->__jmpbuf[__JMP_BUF_SP+1] = (int)(func); /* pc */ \
   buf->__jmpbuf[__JMP_BUF_SP] = (int)(stack); /* sp */ \
